extensions/infinispan-embedded/deployment/src/main/java/io/quarkus/infinispan/embedded/deployment/InfinispanEmbeddedProcessor.java

```diff
@@ -209,4 +210,9 @@ private void addReflectionForName(String className, boolean isInterface, IndexVi
     void configureRuntimeProperties(InfinispanRecorder recorder, InfinispanEmbeddedRuntimeConfig runtimeConfig) {
         recorder.configureRuntimeProperties(runtimeConfig);
     }
+
+    @BuildStep
+    public NativeImageResourceBundleBuildItem nativeImageResourceBundle() {
+        return new NativeImageResourceBundleBuildItem("sun.security.util.Resources");
```

Comment:
I think I would do that somewhere in the core. Not in a specific extension. Given that's a JDK class and there's a good chance we might need this for other extensions.

Reply:
I thought about it too, but since I have no idea of the impact it can have on the native image (in termes of size, memory, etc.), I went for the cautious way. I'll move it to `quarkus-core-deployment`.